.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
@deshraj Please have a look at the comparison of `flake8` and `pylint` [here](https://python.libhunt.com/project/pylint/vs/flake8). Pylint is a good, handy, convenient and wonderful tool to be used.